Update README.md (#367)
solid/solid-panes: README.md

npm run start
```
2. a browser window should automatically open at http://localhost:9000, if for some reason it doesn't go ahead and manually navigate there.

> 3. Once you arrive at the Solid Pane Tester page, the profile-pane will be loaded by default, Proceed to edit solid-panes/dev/loader.ts and at line 5, you should see import Pane from 'profile-pane'. Simply change 'profile-pane' to your preferred pane/directory containing the pane of choice, for example, you could choose 'source-pane' and bam, it will load that pane. For those who are new, you can go to the solid-panes/src directory and manually navigate through each individual folder. In most folders, you simply look for any file that has Pane in the title. Copy and paste the pane.js file of your choice into the solid-panes/dev/pane folder or you can simply import directly from the src directory. For example import from '../src/dokieli/dokieliPane' will work just fine. Each time you save solid-panes/dev/loader.ts while importing a different pane, your browser at http://localhost:9000/ should automatically refresh. It's a good idea to keep the developer console open in your web browser to ensure panes are loading and rendering properly. 

Reviewer: ```suggestion
3. Once you arrive at the Solid Pane Tester page, the `profile-pane` will be loaded by default. Proceed to edit `solid-panes/dev/loader.ts` and, at line 5, you should see `import Pane from 'profile-pane'`. Simply change `'profile-pane'` to your preferred pane/directory containing the pane of choice; for example, you could choose `'source-pane'` and bam, it will load that pane. For those who are new, you can go to the `solid-panes/src` directory and manually navigate through each individual folder. In most folders, you simply look for any file that has `pane` in the title. Copy and paste the `pane.js` file of your choice into the `solid-panes/dev/pane` folder, or you can import directly from the `src` directory. For example, importing from `'../src/dokieli/dokieliPane'` will work just fine. Each time you save `solid-panes/dev/loader.ts` while importing a different pane, your browser at `http://localhost:9000/` should automatically refresh. It's a good idea to keep the developer console open in your web browser to ensure panes are loading and rendering properly. 
```

> 4. Another tip to ensure you arrive at the proper destination, is to look at line 48-53 in solid-panes/dev/loader.ts. You should see an event listener that is ready for a string. renderPane('https://solidos.solidcommunity.net/Team/SolidOs%20team%20chat/index.ttl#this') will be the default. Depending on the pane.js that you choose from the earlier import statements, this renderPane function will determine how you want to see DOMContent inside of that particular pane. If you have created an index.html in your provider pod storage area, you could use 'https://yoursolidname.solidcommunity.net/profile/index.html' inside of the renderPane() function parameters. You can either edit the string manually in solid-panes/dev/loader.ts or you can go to your developer console and type renderPane('https://yoursolidname.solidcommunity.net/profile/index.html') just point to a part of your account that is congruent to the pane that you wish to import! :)

Reviewer: ```suggestion
4. Another tip: to ensure you arrive at the proper destination, look at lines 48–53 in `solid-panes/dev/loader.ts`. You should see an event listener that is ready for a string. `renderPane('https://solidos.solidcommunity.net/Team/SolidOs%20team%20chat/index.ttl#this')` will be the default. Depending on the `pane.js` that you chose in the earlier import statements, the `renderPane` function determines the way you will see DOMContent inside of that particular pane. If you have created an `index.html` in your provider pod storage area, you could use `'https://yoursolidname.solidcommunity.net/profile/index.html'` inside of the `renderPane()` function parameters. You can edit the string manually in `solid-panes/dev/loader.ts`, or you can go to your developer console and type `renderPane('https://yoursolidname.solidcommunity.net/profile/index.html')` — just point to a part of your account that is congruent to the pane that you wish to import! :)
```
